srcpkgs/usbguard/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'usbguard'
+pkgname=usbguard
+version=0.7.5
+revision=1
+archs="~*-musl"
```

Comment:
plse add ```
case "$XBPS_TARGET_MACHINE" in
    *-musl) broken="https://travis-ci.org/void-linux/void-packages/jobs/575570770";;
esac
```
instead. Or try to patch the code to work with the posix version of basename provided in `<libgen.h>`

Reply:
Just checked the code, it should be safe to use the posix version, there is no case where the difference between these two should make a problem.

Reply:
I'll take a look, there seem to be some other incompatibilities too though

Reply:
looks to be working